### PR TITLE
feat(mep): Adding histogram support to metrics enhanced perf

### PR DESCRIPTION
**Pull request by wmak**
- This uses the metrics dataset to supply histogram data in the same
  format discover expects
- Outlier is currently based on p25 and p75, may change to using tags
  later

### CODE REVIEW

File: src/sentry/api/endpoints/organization_events_histogram.py [hunk 1]
```diff
@@ -51,13 +51,21 @@ def get(self, request: Request, organization) -> Response:
         except NoProjects:
             return Response({})
```
Comment (JoshFerge) [inline src/sentry/api/endpoints/organization_events_histogram.py:53]: @wmak does visibility own this endpoint? or is it unowned?

Comment (wmak) [reply src/sentry/api/endpoints/organization_events_histogram.py:53]: Visibility owns this endpoint, I can update codeowners here too goodcatch 🙏

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -112,6 +112,7 @@ build-utils/        @getsentry/owners-js-build
 
 ### Visibility ###
 /src/sentry/api/endpoints/organization_tags.py                  @getsentry/visibility
+/src/sentry/api/endpoints/organization_events_histogram.py			@getsentry/visibility
```
Comment (asottile-sentry) [inline .github/CODEOWNERS:115]:
```suggestion
/src/sentry/api/endpoints/organization_events_histogram.py      @getsentry/visibility
```

looks like the rest of these use spaces

**Review by k-fish — APPROVED** [review]
Some minor comments that might need to be addressed, but looks good 👍

**Pull request merged**